config/pmd/ruleset.xml

```diff
@@ -44,17 +50,14 @@
     <!-- Conflicts with the rule category/java/codestyle.xml/AvoidProtectedMethodInFinalClassNotExtending -->
     <exclude name="CommentDefaultAccessModifier"/>
     <!-- Conflicts with the rule category/java/codestyle.xml/AvoidProtectedMethodInFinalClassNotExtending -->
-    <exclude name="DefaultPackage"/>
-
-
-    <!--<exclude name="BeanMembersShouldSerialize"/>-->
-
     <exclude name="ConfusingTernary"/>
+    <exclude name="UseExplicitTypes"/>
+
+    <!-- FIXME: Disabled because of the high number of current violations-->
+    <exclude name="UnnecessaryFullyQualifiedName"/>
 
-    <!-- printStackTrace is used in the Default Amazon handler in a static clause
-        We can re-enable this rule if we find a way to set a logger there
-    -->
-    <!--<exclude name="AvoidPrintStackTrace"/>-->
+    <!-- Disabled because of violations in generated code -->
+    <exclude name="UnnecessaryImport"/>
```

Comment:
Can we exclude generated in a better way?

Reply:
Found it, pushed a fix now.